services/webhook/tests/test_llm_client.py

```
@@ -1380,3 +1380,70 @@ def fake_post(url, **kwargs):
+
+
+def test_coerce_finding_drops_suggestion_redaction_would_alter() -> None:
+    """#553 audit stage 8: a suggestion that echoed a secret is DROPPED,
+    never rendered - a committable block containing [REDACTED:...] would
+    one-click the placeholder into source."""
+    ok, _ = lc._coerce_finding({
+        "path": "x.py", "line": 1, "rule": "r", "severity": "high",
+        "message": "m",
+        "suggestion": "key = 'AKIAABCDEFGHIJKLMNOP'",
+    })
+    assert ok is not None and ok.suggestion is None
```

Ruff (0.15.20)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)